extensions/security-jpa/deployment/pom.xml

```diff
@@ -82,7 +82,6 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-hibernate-orm-deployment</artifactId>
-            <scope>test</scope>
```

Comment:
Minor but if it's really necessary, we need to move it up with the other non-test dependencies.